File: Programming Basics with Python/While-цикъл/old_books.py

book = input()
x = input()
count = 0

the_book = False
while x != "No More Books":
    if x == book:
        the_book = True
        break
    count += 1
    x = input() 
    
    
if the_book:
    print(f"You checked {count} books and found it.")
else:
    print("The book you search is not here!")
    print(f"You checked {count} books.")

File: Programming Basics with Java/While-loop/OldBooks.java
import java.util.Scanner;

public class OldBooks 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        String book_name = input.nextLine();

        int count = 0;

        while (true)
        {
            String book = input.nextLine();

            if (book.equals(book_name))
            {
                System.out.printf("You checked %d books and found it.", count);
                break;
            }
            else if (book.equals("No More Books"))
            {
                System.out.println("The book you search is not here!");
                System.out.printf("You checked %d books.", count);
                break;
            }

            count += 1;
        }
    }
}
